### PR TITLE
Add assertion attributes to child object on profile (passport-saml#543)

### PR DESCRIPTION
**Pull request by kriss1897**
This attributes are also mounted to profile directly in a non conflicting way.

Duplicate PR of: https://github.com/node-saml/passport-saml/pull/593

### CODE REVIEW

**Comment by kriss1897** [comment]
This failed build on 16.x is also failing on master. Will check and raise a PR to fix

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -24,7 +24,11 @@ jobs:
       - run: npm ci
       - run: npm test
       - run: npm ci
-      - run: npm --depth 9999 update
+      - run: npm update
```
Comment (cjbarth) [inline .github/workflows/workflow.yml:27]: How is this then working in `passport-saml` if it isn't working here in `node-saml`?

**Comment by cjbarth** [comment]
@kriss1897 Do you have a link to the bug in `npm`?

File: .github/workflows/workflow.yml (hunk 1 above)
Comment (kriss1897) [reply .github/workflows/workflow.yml:27]: I tried running these commands in the same sequence with passport-saml's package.json on this repo, but still the command was failing for this repo and not for passport-saml.

**Comment by kriss1897** [comment]
No, I didn't see any open npm tickets regarding the same. I found some old ones and some stackoverflow threads. Try this: https://www.google.com/search?q=cannot+find+module+npmlog+after+npm+update

**Comment by cjbarth** [comment]
It seems that this has been [fixed](https://stackoverflow.com/a/34830277/271351). Could we revert the change and try again?

**Pull request merged**